config/application.yml

```diff
@@ -0,0 +1,2 @@
+stripe_secret_key: "sk_test_OGO6Th4XlLrPTcNjOSyVOfqx"
```

Comment:
This file should be in `.gitignore`. You just published your secret keys and anyone who can read this can abuse it. For that reason confidential information is always under ENV variables and files like this are .gitignored. :) I know the test key is the test key - but you don't want to share real live key this way.